.github/workflows/reusable_general.yml

```
@@ -0,0 +1,28 @@
+---
+name: Reusable Checks that are repo-language agnostic
+
+on:
+  workflow_call:
+    inputs:
+      fedora-container-image-number:
```

actionlint (1.7.12)

[error] "type" is missing at "fedora-container-image-number" input of workflow_call event

(syntax-check)